test/parallel/test-promises-unhandled-rejections.js

```diff
@@ -684,3 +684,16 @@ asyncTest('Throwing an error inside a rejectionHandled handler goes to' +
     }
   }, 1);
 });
+
+asyncTest('Rejected promise inside unhandledRejection allows nextTick loop' +
+          ' to proceed first', function(done) {
+  clean();
+  Promise.reject(0);
+  let didCall = false;
+  process.on('unhandledRejection', () => {
+    assert(!didCall);
+    didCall = true;
```

Comment:
I’m not sure, but, `common.mustCall()`? ;)

Reply:
Since it checks at `exit`, this would just keep looping forever until timeout is hit.